Earthfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.10
```

Comment:
Should we use alpine for faster builds?

Reply:
I wanted to, but protoc requires `__strftime_l` which isn't included under alpine's glibc lib, so I ended up using ubuntu which is definitely more bloated but got me working faster.

Reply:
:-( makes sense